src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -447,6 +448,12 @@ def add_summary_full(self, key, refcase) -> EnkfConfigNode:
 
     def addNode(self, config_node: EnkfConfigNode):
         assert isinstance(config_node, EnkfConfigNode)
+        assert config_node is not None
+        key = config_node.getKey()
+        if key in self:
+            raise ConfigValidationError(
+                f"Enkf config node with key {key} " "already present in ensemble config"
```

Comment:
Since you already have to rebase, this splitting of the string seems to not be necessary.

Reply:
good catch, thanks!